src/Bundle/Controller/Parameters.php

```diff
@@ -20,7 +20,7 @@ class Parameters extends ParameterBag
     /**
      * {@inheritdoc}
      */
-    public function get($path, $default = null)
+    public function get(string $path, $default = null)
```

Comment:
Is it required? It's a BC break unfortunately.

Reply:
for psalm only, but I can suppress this psalm warning.